Request to add simplified Chinese Translation

Augustakit opened the pull request:
I updated the simplified Chinese translation a few days ago. Please merge it

diwako commented on addons/main/stringtable.xml:
```diff
@@ -301,22 +336,23 @@
             <German>Basisreichweite über die unabhängige und civile Einheiten Informationen teilen</German>
             <Polish>Podstawowy zasięg dzielenia się informacjami dla strony INDEPENDENT</Polish>
             <Russian>Расстояние на котором независимая и гражданская стороны делятся информацией</Russian>
-            <Italian>Le parti indipendenti e civili del raggio di base condividono le informazioni</Italian>
```
Do not remove existing entries

diwako commented on addons/main/stringtable.xml:
```diff
@@ -301,22 +336,23 @@
             <German>Basis Unabhängige</German>
             <Polish>Podstawa INDEPENDENT</Polish>
             <Russian>Base Independent</Russian>
-            <Italian>Indipendente dalla base</Italian>
```
Do not remove existing entries

diwako reviewed (changes requested):
Make sure to leave entries of other languages.

diwako commented:
Reported issues from the linting

```
Checking addons/main/stringtable.xml:
  ERROR: Incorrect number of indenting spaces on line 71, currently 13, should be 12.
  ERROR: Incorrect number of indenting spaces on line 89, currently 13, should be 12.
  ERROR: Incorrect number of indenting spaces on line [14](https://github.com/nk3nny/LambsDanger/runs/6100350623?check_suite_focus=true#step:4:14)3, currently 13, should be 12.
  ERROR: Incorrect number of indenting spaces on line [19](https://github.com/nk3nny/LambsDanger/runs/6100350623?check_suite_focus=true#step:4:19)5, currently 13, should be 12.
  ERROR: Incorrect number of indenting spaces on line 204, currently 13, should be 12.
  ERROR: Incorrect number of indenting spaces on line 213, currently 13, should be 12.
```

```
addons/danger/stringtable.xml:
	396: Trailing whitespace
addons/main/stringtable.xml:
	71: Wrong amount of left-padding spaces(want multiple of 4)
	89: Wrong amount of left-padding spaces(want multiple of 4)
	143: Wrong amount of left-padding spaces(want multiple of 4)
	195: Wrong amount of left-padding spaces(want multiple of 4)
	204: Wrong amount of left-padding spaces(want multiple of 4)
	213: Wrong amount of left-padding spaces(want multiple of 4)
addons/wp/stringtable.xml:
	192: Trailing whitespace
	686: Trailing whitespace
	819: Wrong amount of left-padding spaces(want multiple of 4)
	1221: Trailing whitespace
```

Augustakit commented:
I have fixed all the problems you pointed out. Please help me check them. Thank you very much

diwako commented on addons/wp/stringtable.xml:
```diff
@@ -255,6 +286,7 @@
             <Polish>Promień</Polish>
             <Russian>Радиус</Russian>
             <Italian>Raggio</Italian>
+            <Chinesesimp>建筑和静态武器半径范围内驻守并配备/n两倍于该数字的人员用于巡逻</Chinesesimp>
```
I only google translated this to check as this looked rather long. It seems this is the tooltip of "TaskCamp"_Radius and not the display name.

The following entries are also not correct as they represent the next entry in the string table.

Augustakit commented:
Sorry. I'm so very sorry about that one.I accidentally uploaded some translations at the wrong time, which led you to see such results, but I have corrected them now, and checked the pages of other translation documents and corrected them again.
